.goreleaser.yaml

```
@@ -34,7 +34,14 @@ builds:
       #   goarch: arm
 
     flags:
-    - -tags=b_tiny,b_norepl
+    - -tags=b_tiny
```

eg. with the ultimate bloat working set of tags: 😄
```suggestion
    - -tags=b_sqlite,b_http,b_sql,b_postgres,b_bson,b_crypto,b_smtpd,b_mail,b_bcrypt,b_telegram,b_html,b_contrib,b_openai,b_email,b_mail,b_mysql,b_nats,b_psql,b_psutil,b_sxml,b_echo
```